milabench/cli.py

```diff
@@ -318,9 +338,10 @@ def run():
         }[dash]
 
         mp = get_multipack(run_name=run_name)
+        arch = next(iter(mp.packs.values())).config["system"]["arch"]
 
         # Initialize the backend here so we can retrieve GPU stats
-        init_arch()
+        init_arch(arch)
```

Comment:
This is a bit convoluted way to retrieve and set `arch` so let me know if you don't like it

Reply:
Humm, it will work for now